website/src/documentation/programming-guide.md

```diff
@@ -129,6 +129,11 @@ Pipeline p = Pipeline.create(options);
 {% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py tag:pipelines_constructing_creating
 %}
 ```
+```go
+// In order to start creating the pipeline for execution, a Pipeline object is needed.
+p := beam.NewPipeline()
```

Comment:
Consider swapping with the single line convenience function:
 p, s := beam.NewPipelineWithRoot()